include/utilities/realtime/dataman/DataMan.h

```diff
@@ -0,0 +1,468 @@
+#ifndef DATAMAN_H_
+#define DATAMAN_H_
+
+#include <chrono>
+#include <cstdint>
+#include <dlfcn.h>
+#include <functional>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <unistd.h>
+#include <vector>
+
+#include "external/json.hpp"
+
```

Comment:
Try to logically group your includes and then sort within the groups.  Something like:
```
// std c headers
// system c headers
// std c++ headers
// other package headers
```
So in this case, it would look like
```
#include <cstdint>

#include <dlfcn.h>
#include <unistd.h>

#include <chrono>
#include <functional>
#include <iostream>
#include <memory>
#include <string>
#include <vector>

#include "external/json.hpp"
``` 

And once the implementation of members is moved to a seperate cpp file, do `dlfcn.h` and `unistd.h` need to be in the public header?  Try to move them to the cpp file as well if possible.